pkg/cmd/attestation/inspect/inspect.go

```diff
@@ -0,0 +1,166 @@
+		Use:   "inspect [<file path> | oci://<OCI image URI>] --bundle <path-to-bundle>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Inspect a sigstore bundle",
+		Long: heredoc.Docf(`
+			Inspect a downloaded Sigstore bundle for a given artifact.
+				
+			The command requires either:
+			* a relative path to a local artifact, or
+			* a container image URI (e.g. %[1]soci://<my-OCI-image-URI>%[1]s)
+
+			Note that if you provide an OCI URI for the artifact you must already
+			be authenticated with a container registry.
+
+			The command also requires the %[1]s--bundle%[1]s flag, which provides a file
+			path to a previously downloaded Sigstore bundle. (See also the %[1]sdownload%[1]s
+			command).
+
+			By default, the command will print information about the bundle in a table format.
+			If the %[1]s--json-result%[1]s flag is provided, the command will print the 
+			information in JSON format.
+		`, "`"),
+		Example: heredoc.Doc(`
+			# Inspect a Sigstore bundle and print the results in table format
+			$ gh attestation inspect <my-artifact> --bundle <path-to-bundle>
+
+			# Inspect a Sigstore bundle and print the results in JSON format
+			$ gh attestation inspect <my-artifact> --bundle <path-to-bundle> --json-result
+
+			# Inspect a Sigsore bundle for an OCI artifact, and print the results in table format
+			$ gh attestation inspect oci://<my-OCI-image> --bundle <path-to-bundle>
+		`),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			// Create a logger for use throughout the inspect command
+			opts.Logger = logging.NewDefaultLogger(f.IOStreams)
+
+			// set the artifact path
+			opts.ArtifactPath = args[0]
+
+			// Check that the given flag combination is valid
+			if err := opts.AreFlagsValid(); err != nil {
+				return err
+			}
+
+			// Clean file path options
+			opts.Clean()
+
+			return nil
+		},
+		// Use Run instead of RunE because if an error is returned by RunInspect
+		// when RunE is used, the command usage will be printed
+		// We only want to print the error, not usage
+		Run: func(cmd *cobra.Command, args []string) {
+			opts.OCIClient = oci.NewLiveClient()
+
+			if err := auth.IsHostSupported(); err != nil {
+				opts.Logger.Println(opts.Logger.ColorScheme.Red(err.Error()))
+				os.Exit(1)
+			}
+			if err := RunInspect(opts); err != nil {
+				opts.Logger.Println(opts.Logger.ColorScheme.Redf("Failed to inspect the artifact and bundle: %s", err.Error()))
+				os.Exit(1)
+			}
+		},
+	}
+
+	inspectCmd.Flags().StringVarP(&opts.BundlePath, "bundle", "b", "", "Path to bundle on disk, either a single bundle in a JSON file or a JSON lines file with multiple bundles")
+	inspectCmd.MarkFlagRequired("bundle") //nolint:errcheck
+	inspectCmd.Flags().StringVarP(&opts.DigestAlgorithm, "digest-alg", "d", "sha256", "The algorithm used to compute a digest of the artifact (sha256 or sha512)")
+	inspectCmd.Flags().BoolVarP(&opts.JsonResult, "json-result", "j", false, "Output inspect result as JSON lines")
```

Comment:
Use case for JSON output for this command is to pass the output to another tool like jq.

Reply:
It might make sense to model this after the project commands then which provide `--format json` and the utilities that provide that also provide the `--jq` and `--template` flags.

See: https://github.com/cli/cli/blob/dc804d928714120a3f4b53f78847aec7ba282c63/pkg/cmdutil/json_flags.go#L113